DP02_04b_Intermediate_Butler_Queries.ipynb

```diff
@@ -9,8 +9,8 @@
     "<img align=\"left\" src = https://project.lsst.org/sites/default/files/Rubin-O-Logo_0.png width=250 style=\"padding: 10px\" alt=\"Rubin Observatory logo, a graphical representation of turning stars into data.\">\n",
```

Comment:
<p>A more preferred way to get the same information would be:</p>

<pre class="ql-syntax" spellcheck="false">print(butler.dimensions['detector'].schema)
</pre><p> </p><p>That will also include documentation strings (for some fields anyway -- there might be a bug there.)</p><p> </p><p>The <code>RecordClass</code> thing is sort of a weird implementation detail that should probably be private but isn't for some reason -- better not to advertise it. Jim has been adding better ways to get at dimension information over the last 6-12 months.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/211/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Thanks for the clarification!</p>